.github/workflows/deploy.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy
+
+on: push
+
+env:
+  DOCKER_IMAGE: ${{ github.repository }}
+  DOCKER_REGISTRY: ghcr.io
+  DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+  DOCKER_USERNAME: ${{ github.repository_owner }}
+  HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Branch Tag
+        uses: nimblehq/branch-tag-action@v1.2
+
+      - name: Set HEROKU_APP
+        run: |
+          if [[ $BRANCH_TAG = "latest" ]]
+          then
+            echo "::set-env name=HEROKU_APP::go-google-scraper"
+          else
+            echo "::set-env name=HEROKU_APP::go-google-scraper-staging"
+          fi
+
+      - name: Docker login to docker container registry
+        run: "echo $DOCKER_TOKEN | docker login $DOCKER_REGISTRY -u $DOCKER_USERNAME --password-stdin"
```

Comment:
I recommend using [`docker/login-action`](https://github.com/docker/login-action) as it adds a nice DSL while removing some bash script (which is harder to maintain over time)

Reply:
Fixed in d03104d446efd67a72ce26731708bdd1bb4a7e06